MobileCenter/MobileCenterTests/MSAppDelegateForwarderTests.m

```diff
@@ -606,7 +610,9 @@ - (void)testDontForwardSelectorsNotToOverrideIfAlreadyImplementedByOriginalDeleg
 }
 #endif
 
-#if !TARGET_OS_OSX
+#if TARGET_OS_IOS
+
+// TODO: tvOS doesn't support application:openURL:sourceApplication:annotation:. Temporaily disable the test.
```

Comment:
There is no indication that tvOS will ever have a proper "Safari", I don't think we need those around.